CMakeLists.txt

```
@@ -123,6 +123,13 @@ if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux") # uname -s
   endif()
 endif()
 
+if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin") # uname -s
+  if ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "x86_64|arm64") # uname -m
+    set(UTHREAD ON)
+    message("-- Uthread on")
+  endif()
+endif()
+
```

这里逻辑和上面合并下吧，不然一眼容易看错
```suggestion
if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux" or  "${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
```

---

m1 Mac uname -m输出是arm64，Linux下是aarch64？还有就是Mac还没有写ppc架构，逻辑上这两不等同